frontend/src/hooks/useRouterFilter.ts

```diff
@@ -21,7 +21,15 @@ import { PropertyTypes } from 'constants/propertyTypes';
 const extractProps = (props: string[], source: any): any => {
   var dest = {} as any;
   props.forEach(p => {
-    if (source[p] !== undefined) dest[p] = source[p];
+    if (source[p] !== undefined) {
```

Comment:
'true' vs true was causing the filter .equals function to fail.

If we keep having problems with this will need to implement whitelist style comparison on the filter to avoid new additions breaking the comparison.